packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -265,7 +267,22 @@ export class Combobox
 
   @state() selectedVisibleChipsCount = 0;
 
-  //#endregion
+  @state()
+  get allSelected(): boolean {
+    return this.selectedItems.length === this.items.length;
+  }
+
+  @state()
+  get indeterminate(): boolean {
+    return this.selectedItems.length > 0 && this.selectedItems.length !== this.items.length;
+  }
+
+  @state()
+  get visibleKeyboardNavItems(): HTMLCalciteComboboxItemElement["el"][] {
+    return this.keyboardNavItems().filter((item) => !isHidden(item));
```

Comment:
Can you move the `keyboardNavItems()` into a stateful getter? Otherwise this state won't change because this is a function, not another stateful property that would cause `visibleKeyboardNavItems` to update.